core/src/main/resources/lib/form/select/select.js

```diff
@@ -43,6 +43,9 @@ function updateListBox(listBox,url,config) {
 Behaviour.specify("SELECT.select", 'select', 1000, function(e) {
 
         function hasChanged(selectEl, originalValue) {
+            // seems like a race condition allows this to fire before the 'selectEl' is defined. If that happens, exit..
+            if(!selectEl || !selectEl.options || !selectEl[0]) 
```

Comment:
assume this should be:
if(!selectEl || !selectEl.options || !selectEl.options[0])